crates/bevy_winit/src/winit_windows.rs

```diff
@@ -207,15 +196,20 @@ impl WinitWindows {
             display_handle: winit_window.raw_display_handle(),
         };
         self.windows.insert(winit_window.id(), winit_window);
-        Window::new(
+        let mut window = Window::new(
             window_id,
             window_descriptor,
             inner_size.width,
             inner_size.height,
             scale_factor,
             position,
             Some(raw_handle),
-        )
+        );
+        // Do not set the grab mode on window creation if it's none, this can fail on mobile
+        if window_descriptor.cursor_grab_mode != CursorGrabMode::None {
+            window.set_cursor_grab_mode(window_descriptor.cursor_grab_mode);
```

Comment:
I think I would prefer if this still logged an error here, so the user gets some feedback rather than silently failing

Reply:
Sorry for the late reply. I'm experiencing covid-19 this week. 

Like the fallback will be done in `bevy_winit::change_window`, the errors, if any, will also be logged there: https://github.com/bevyengine/bevy/blob/1aeaafa7c455163d1f2ef59d9404a24a00c5b138/crates/bevy_winit/src/lib.rs#L150.

Reply:
oh gotcha nvm, my brain was in windows as entities mode and I forgot we had window commands